R/distribution_formulas.R

```diff
@@ -264,7 +264,11 @@ get_expected_name <- function(family, data_type) {
 initialize_data_distribution <- function(
     module,
     family = NULL,
-    sd = list(value = 1, estimation_type = "constant"),
+    # Create a tibble with value and estiamtion_type column for sd
+    sd = dplyr::tibble(
```

Comment:
We have {tibble} as an import if you want to use the direct function instead of the exported one.